src/less/components/badge.less

```diff
@@ -3,8 +3,41 @@
 /* stylelint-disable */
 @badge-list: {
     default: @brand-primary;
-    inactive: @brand-bg-gray;
+    inactive: @list-gray;
     warning: @brand-error;
+    yellow: @brand-primary;
+    yellow-light: @brand-primary-light;
+    yellow-lighter: @brand-primary-lighter;
+    red: @brand-error;
+    red-dark: @brand-error-dark;
+    red-light: @brand-error-light;
+    white: @brand-bg-white;
+    pink: @brand-pink;
+    brown: @brand-brown;
+    brown-medium: @brown-medium;
+    brown-solid: @solid-brown;
+    brown-soft: @brown-soft;
+    brown-dark: @dark-brown;
+    brown-light: @brown-light;
+    turqouise: @brand-turquoise;
+    turqouise-dark: @brand-turqouise-dark;
+    turqouise-darker: @brand-turqouise-darker;
+    turqouise-light: @brand-turquoise-light;
+    turqouise-lighter: @brand-turqouise-lighter;
+    apricot: @brand-apricot;
+    gray: @list-gray;
+    gray-dark: @brand-bg-gray;
+    gray-light: @brand-bg-gray;
+    orange: @brand-orange;
+    orange-light: @brand-warning-light;
+    green: @brand-green;
+    green-light: @brand-green-light;
+    blue: @brand-blue;
+    blue-light: @brand-blue-light;
+
+    // Should be reworked to variables containing both text color and bg color?
+
+    // Contrast checker -> https://webaim.org/resources/contrastchecker/
```

Comment:
I guess you have validated the contrasts? 🤓

Reply:
Yes, I just pasted the one i used 😎